tests/virtualization/yast_virtualization.pm

```diff
@@ -60,10 +61,10 @@ sub run() {
     send_key "alt-f10";
     wait_idle;
     become_root;
-    type_string "systemctl start libvirtd", 50;
+    systemctl 'systemctl start libvirtd', 50;
```

Comment:
This will result in a command like this: "systemctl --no-pager systemctl start libvirtd"